components/UnitCsvViewer.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div v-if="loading">Loading</div>
+  <v-data-table v-else v-bind="{ ...csvContent }" multi-sort />
```

Comment:
Could be better to use the component UnitQueryResults instead of a plain v-data-table. This would provide a filter bar.

Note: in this case we would rename UnitQueryResults into something more fitting (e.g. UnitFilteredTable)

Reply:
> From my understanding we want to be able to put files from **any** source, even if no card was created for it. So we would still need to add a **separate** card on the homepage, called "Generic viewer" or something. It would not contain any experience, just the ability to upload a zip and browse the files with this viewer.

Yes, that's what I understand as well. In this case, the problem is that each experience "knows" what files to look for, and currently if this is not specified the code crashes.

Reply:
> Could be better to use the component UnitQueryResults instead of a plain v-data-table. This would provide a filter bar.
> 
> Note: in this case we would rename UnitQueryResults into something more fitting (e.g. UnitFilteredTable)

Yes indeed that seems like a good idea, although I think there is still benefit to having a dedicated `UnitCsvViewer` component, as it avoids complexifying `UnitFileExplorer`